packages/libs/react-ui/.babelrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "sourceType": "unambiguous",
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "chrome": 100
```

Comment:
to be honest, I'm not sure how this was selected - I copied it from the other component library. Do you have recommendations for targets or would you suggest to remove them?

Reply:
So it looks like most of the apps don't set it, only the react-ui, react-components, and kadena-docs. I'm leaning towards not setting it and using the default since it doesn't seem like we have a good reason to make it more specific. What do you think @alber70g and @webpro?

Reply:
Yeah let's leave it out then :)